app/helpers/remote_execution_helper.rb

```diff
@@ -236,4 +241,21 @@ def load_template_from_task(template_invocation, target)
     return if task.nil?
     task.execution_plan.actions[1].try(:input).try(:[], 'script')
   end
+
+  def jobs_template_with_input(job_invocation)
+    template = ReportTemplate.where(name: 'Job Invocation report').first
+    input_id = template.template_inputs.where(name: 'job_id').first.id
```

Comment:
Template might have been renamed or deleted and `template_inputs` is not defined for nil

Reply:
Perhaps this should be configurable through settings, like ansible inventory template. That prevents hardcoding the name. But yeah, we need to be more defensive here.